README.md

```diff
@@ -313,6 +313,15 @@ bad data ends up in our database. We also discussed the difference between model
 validations and database constraints. Finally, we saw some common methods for
 implementing validations on our models using Active Record.
 
+## Check For Understanding
+
+Before you move on, make sure you can answer the following questions:
+
+1. When would you use the `validates` method vs. the `validate` method? What
+   argument(s) get passed to each of them?
+2. What are the two ways to trigger validation?
```

Comment:
There are more than two ways to trigger validations: https://guides.rubyonrails.org/active_record_callbacks.html#running-callbacks

Suggested change (remove _the_):

2. What are two ways to trigger validation?